### PR TITLE
Add UpdateNodeCollectors message

### PR DESCRIPTION
**Pull request by MrZammler**
Adds a new message `UpdateNodeCollectors` which will contain a list of modules and plugins from an agent instance.

### CODE REVIEW

File: proto/nodeinstance/info/v1/info.proto [hunk 1]
```diff
@@ -124,3 +125,24 @@ message MachineLearningInfo {
...
+//
+// key: `claim_id,node_id`
+//
+// Publishers: `netdata/agent`
+//
+// When:
+//  On nodeinstance connect (after agent settles) and on detection of change of collectors
+//
+
+message collectorinfo {
```
Comment (underhood) [inline proto/nodeinstance/info/v1/info.proto:139]: PascalCaseIt?

Comment (MrZammler) [reply proto/nodeinstance/info/v1/info.proto]: Ah yes :-)

**Review by underhood — APPROVED** [review]

**Review by vkuznecovas — APPROVED** [review]

**Comment by vkuznecovas** [comment]
A couple of questions in regard to this when do you guys intend to release this? 

Also, when will you guys be emitting these events? As depending on your answer to this we might need to have changes on the cloud prior to you guys releasing this to the agent.

**Comment by MrZammler** [comment]
Hi Viktoras!

I've been checking, let's not merge it before we test and see if we're ok?

I have something to send this message, just to check if you guys are ok receiving it. We can test that. Not sure if it requires a new topic though.

As to when we are going to emit these events, ideally when there's changes to collectors. We'll try to group the initial adding of collectors (i.e. when an agent starts, or children connect) to a single message. Then I guess we'll see when again will be sent, most likely when a collector is stopped. I will need to check how often this happens, and again to minimize the messages sending until things settle.

**Comment by vkuznecovas** [comment]
> I have something to send this message, just to check if you guys are ok receiving it. We can test that. Not sure if it requires a new topic though.

I'll probably ping you next week then, as with AWS migration and the office visit I'm unlikely to add this until then.

**Pull request merged**